app/controllers/finders_controller.rb

```diff
@@ -17,6 +19,8 @@ def show
       format.html do
         raise UnsupportedContentItem unless content_item.is_finder?
 
+        @show_banner = show_banner?(request.path)
```

Comment:
The base path could be used here as the controller has access to the [content item](https://github.com/alphagov/finder-frontend/blob/7cd96b16dcbe1f8a27fdc79dcb2313bdd5e8dedd/app/controllers/application_controller.rb#L57).